app/assets/javascripts/miq_application.js

```diff
@@ -1246,6 +1246,7 @@ function miqInitSelectPicker() {
     style: 'btn-info',
     size: 4
   });
+  $('[title]').not('.selectpicker').tooltip({container: 'none'});
```

Comment:
Is this saying that all select pickers will have tooltips removed?  Is that what is intended?